### PR TITLE
Use cert-controller to generate webhook certificates

### PR DESCRIPTION
**Pull request by astefanutti**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -127,34 +130,52 @@ func main() {
...
 
 	OpenShift := isOpenShift(ctx, kubeClient.DiscoveryClient)
 
-	if OpenShift {
-		// TODO: setup the RayCluster webhook on vanilla Kubernetes
-		exitOnError(controllers.SetupRayClusterWebhookWithManager(mgr, cfg.KubeRay), "error setting up RayCluster webhook")
-	}
+	go setupControllers(mgr, kubeClient, cfg, OpenShift, certsReady)
+
+	exitOnError(mgr.AddHealthzCheck(cfg.Health.LivenessEndpointName, healthz.Ping), "unable to set up health check")
```
Comment (dgrove-oss) [inline main.go:164]: Do you want to grab the code block from the AppWrapper controller that defers the health/readiness probes until the certificates or ready?  I called it in #491, but probably best to copy it to the codeflare operator's main.  Code is here: https://github.com/project-codeflare/appwrapper/blob/2213dec59688ddc2259f2d0c5cea4ad759b0e047/pkg/controller/setup.go#L96-L112

Comment (astefanutti) [reply main.go]: Right, that's a good idea. Added in af8f912f2218f3a1356a880dbc44af8c592950e1.

**Review by sutaakar — COMMENTED** [review]
/lgtm

**Review by dimakis — APPROVED** [review]

**Pull request merged**